docs/flatcar-linux/kvm-libvirt.md

```diff
@@ -0,0 +1,297 @@
+# KVM libvirt
+
+In this tutorial, we'll create a Kubernetes cluster with Flatcar Linux inside VMs on our local machine.
+
+We'll declare the cluster using the Lokomotive Terraform module for the KVM libvirt platform.
+
+Controllers are provisioned to run an `etcd-member` peer and a `kubelet` service. Workers run just a `kubelet` service. A one-time [bootkube](https://github.com/kubernetes-incubator/bootkube) bootstrap schedules the `apiserver`, `scheduler`, `controller-manager`, and `coredns` on controllers and schedules `kube-proxy` and `calico` (or `flannel`) on every node. A generated `kubeconfig` provides `kubectl` access to the cluster.
+
+## Requirements
+
+* free RAM (minimum 4 GB)
```

Comment:
I would rephrease to "At least 4 GB of free RAM". In any case the first letter in the sentence should be capitalized, unless it is a special term which is normally written in lowercase (e.g. `libvirt`).

Reply:
Thanks for the feedback. Yes, I've included your suggestions to make things more clear.